From 0bd84872e18d4183310ef44ac790cbb0861d51e0 Mon Sep 17 00:00:00 2001
From: Andreas Brandl <abrandl@gitlab.com>
Date: Tue, 29 Dec 2020 23:24:20 +0100
Subject: [PATCH] Disable automatic schema dumping after migration

---
 config/environments/test.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/environments/test.rb b/config/environments/test.rb
index 4e359cd308c..0a08c1b7b98 100644
--- a/config/environments/test.rb
+++ b/config/environments/test.rb
@@ -50,6 +50,8 @@
 
   config.active_job.queue_adapter = :test
 
+  config.active_record.dump_schema_after_migration = false
+
   if ENV['CI'] && !ENV['RAILS_ENABLE_TEST_LOG']
     config.logger = ActiveSupport::TaggedLogging.new(Logger.new(nil))
     config.log_level = :fatal
-- 
2.29.0

