From 47984da16be2db9bc307b74a7e5d2f400b1c24c0 Mon Sep 17 00:00:00 2001
From: Andreas Brandl <abrandl@gitlab.com>
Date: Wed, 24 Feb 2021 15:44:25 +0100
Subject: [PATCH] Disable schema dump

---
 config/environments/development.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/environments/development.rb b/config/environments/development.rb
index 50d394859bc..38c69fbe205 100644
--- a/config/environments/development.rb
+++ b/config/environments/development.rb
@@ -71,4 +71,6 @@

     config.middleware.delete BetterErrors::Middleware
   end
+
+  config.active_record.dump_schema_after_migration = false
 end
--
2.30.0
